Sources/TuistCore/Utils/System.swift

```diff
@@ -159,6 +159,8 @@ public enum SystemError: FatalError {
     public var type: ErrorType { return .abort }
 }
 
+public var verbose = false
```

Comment:
I'd avoid global variables like this one.

Reply:
I will just remove this and the verbose logging, it was super useful when trying to work out what was going on. But to implement this properly requires a lot more work and it's not something I would want to do in this P/R.

Reply:
I have removed all verbose logic. For reference, all the code was removed in https://github.com/tuist/tuist/pull/394/commits/25a7108c8ed4680984809eed55821cdc70f206a1